src/disk.ts

```diff
@@ -60,7 +60,7 @@ export async function mountRamDisk(size: number, name: string = generateDriveNam
         }
       });
     });
-    process = exec(`sudo mount -t tmpfs -o size=${size}m tmpfs ${mountPath}`);
+    process = exec(`sudo mount -t tmpfs -o size=${size} tmpfs ${mountPath}`);
```

Comment:
Could we use `execa` as it does not leak children, and comes pre-baked with a promise interface.

Reply:
Good call, done